packages/core/http/core-http-server/src/router/route_validator.ts

```diff
@@ -79,7 +79,7 @@ export type RouteValidationFunction<T> = (
  *
  * @public
  */
-export type RouteValidationSpec<T> = ObjectType | Type<T> | RouteValidationFunction<T>;
+export type RouteValidationSpec<T> = ObjectType | Type<unknown, T> | RouteValidationFunction<T>;
```

Comment:
This enables the route handlers to receive the transformed output type instead of the regular type.